### ISSUE TITLE
Style/MutableConstant on heredocs

### ISSUE THREAD

**Issue by dkniffin** [bbatsov/rubocop#5725]
## Expected behavior

Rubocop errors about style/MutableConstant for heredocs

## Actual behavior

Either it should not error, or there should be documentation on how to handle that case

## Steps to reproduce the problem

With the default rubocop config, run the linter on the following:

```rb
TESTING = <<TESTING
  content
TESTING
```

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.52.1
```

**Comment by mikegee** [comment]
> ... or there should be documentation ...

Would you consider the change made by Rubocop's autocorrect documentation?

```ruby
TESTING = <<TESTING.freeze
  content
TESTING
```

**Comment by dkniffin** [comment]
@mikegee That's exactly what I was looking for. I had no idea how to freeze a heredoc, and I didn't see it here: https://github.com/bbatsov/rubocop/blob/master/manual/cops_style.md#stylemutableconstant

**Comment by mikegee** [comment]
Yeah, the syntax for heredocs can be awkward.
